### PR TITLE
Refactors for services

### PR DESCRIPTION
**Pull request by adbarc92**

### CODE REVIEW

File: src/db/buildTables.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	id SERIAL PRIMARY KEY,
+  answer_id INT NULL DEFAULT NULL,
+  photo_url TEXT NULL DEFAULT NULL
+);
+
+INSERT INTO qa.qa_photos (answer_id, photo_url) SELECT answer_id, photo_url FROM temp_photos;
+
+DROP TABLE temp_photos;
+
+select row_to_json(art) as artists
```
Comment (jordanabbasi) [inline src/db/buildTables.sql]: remove / comment out

File: src/db/index.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Pool, PoolClient } from 'pg';
+
+const connection = new Pool({
+  user: 'postgres',
+  host: 'localhost', // * This will inevitably change
+  database: 'questions_and_answers',
+  password: 'postgres',
+  port: 5432,
+	max: 50,
```
Comment (jordanabbasi) [inline src/db/index.ts:9]: remove extra indentation here

File: scripts/createDB.bash [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# su - postgres
+dropdb 'questions_and_answers'
+createdb 'questions_and_answers'
+psql questions_and_answers
+# \i ../src/db/createTables.sql
```
Comment (jordanabbasi) [inline scripts/createDB.bash]: add newline at end

File: src/db/indexing.sql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+CREATE INDEX idx_product_id ON qa.questions(product_id);
+CREATE INDEX idx_question_id ON qa.answers(question_id);
+CREATE INDEX idx_answer_id ON qa.qa_photos(answer_id);
```
Comment (jordanabbasi) [inline src/db/indexing.sql]: add newline at end

File: src/index.ts [hunk 5]
```diff
@@ -18,97 +17,196 @@ app.use(express.urlencoded({ extended: true }));
 
 /* Questions List */
 app.get('/qa/questions', async (req: Request, res: Response) => {
-  console.debug('req.url:', req.url);
-  let client;
-
   try {
+    console.debug('req.url:', req.url);
```
Comment (jordanabbasi) [inline src/index.ts]: remove?

**Review by jordanabbasi — APPROVED** [review]

**Review by alishaedington — APPROVED** [review]
Looks Good.

**Pull request merged**